tests/test_cli.py

```diff
@@ -96,6 +98,24 @@ def _parse_output(output: str, output_format: str) -> Any:
         assert False
 
 
+def _get_output(command: list[str], output_format: str) -> Any:
+    output = str(subprocess.check_output(command).decode().strip())
+    ret = _parse_output(output, output_format)
+
+    # Since keys are not given in value format, it checks matching with the output in table format.
+    if output_format == "value":
+        table_command = copy.copy(command)
```

Comment:
```suggestion
        # NOTE(nabenabe): We cannot use this function for `test_ask_XXX` because this part creates
        # another trial, making the output different.
        table_command = copy.copy(command)
```